src/test/java/seedu/mtracker/console/AddStockParserTest.java

```diff
@@ -76,6 +76,7 @@ void testStockParameters(String input, String[] expectedStockParameters) throws
         AddStockParser testStockParser = new AddStockParser();
         verifyInstrumentParameters(testStockParser, expectedStockParameters);
     }
+    //@@author

```

Comment:
I think the testXYZParameters method for the addXYZParserTest were written by us also.

Reply:
ok